### ISSUE TITLE
View [0] not found. ../vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php line 137

### ISSUE THREAD

**Issue by alaatarek95** [octobercms/october#3371]
on opening mail templates in settings it should return all templates 

but actually  it shows this error in this project only ,, I don't know why I have changed the vendor and modules files but I still receiving this error ,, I tried to enter new mail templates manualy to the database table system_mail_templates but I recieved error while sending mail by them saying (Unknown "partial" tag in "__string_template__c850328ca597a3b3316e78bbca587192fe87597de54477386adefaf0af606191)
I cleared the cash but still receiving this error and mail templates view still not working 

I have post this issue on octobercms support but till now nobody answered me I don't know where to get it solved so I a posting it here 
```php          
  if ($this->files->exists($viewPath = $path.'/'.$file)) {
                return $viewPath;
            }
        }
    }

    throw new InvalidArgumentException("View [$name] not found.");
}
```
415

**Comment by LukeTowers** [comment]
@alaatarek95 did you install via composer or the installer?

**Comment by alaatarek95** [comment]
@LukeTowers via installer

**Comment by LukeTowers** [comment]
Have you disabled core updates? Can you upgrade to the latest build? It'll require PHP 7

**Comment by alaatarek95** [comment]
it works! thank you @LukeTowers

**Comment by Shara96** [comment]
I have php 7 for me I can't solve this problem Why?

**Comment by LukeTowers** [comment]
@Shara96 what is your exact error message and stack trace?

**Comment by mohammadmojrian** [comment]
i have same problem with same config ,
i test on php 7 and 5.6

```
exception 'InvalidArgumentException' with message 'View [0] not found.' in /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137
Stack trace:
#0 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('0', Array)
#1 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/View/Factory.php(151): Illuminate\View\FileViewFinder->find('0')
#2 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(215): Illuminate\View\Factory->make(0)
#3 /var/www/irtrip.com/irtrip-b2b/modules/system/models/MailTemplate.php(122): Illuminate\Support\Facades\Facade::__callStatic('make', Array)
#4 /var/www/irtrip.com/irtrip-b2b/modules/system/models/MailTemplate.php(122): Illuminate\Support\Facades\View::make(0)
#5 /var/www/irtrip.com/irtrip-b2b/modules/system/models/MailTemplate.php(90): System\Models\MailTemplate::getTemplateSections(0)
#6 /var/www/irtrip.com/irtrip-b2b/modules/system/controllers/MailTemplates.php(45): System\Models\MailTemplate::syncAll()
#7 [internal function]: System\Controllers\MailTemplates->index()
#8 /var/www/irtrip.com/irtrip-b2b/modules/backend/classes/Controller.php(361): call_user_func_array(Array, Array)
#9 /var/www/irtrip.com/irtrip-b2b/modules/backend/classes/Controller.php(257): Backend\Classes\Controller->execPageAction('index', Array)
#10 /var/www/irtrip.com/irtrip-b2b/modules/backend/classes/BackendController.php(95): Backend\Classes\Controller->run('index', Array)
#11 [internal function]: Backend\Classes\BackendController->run('system/mailtemp...')
#12 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(256): call_user_func_array(Array, Array)
#13 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(164): Illuminate\Routing\Controller->callAction('run', Array)
#14 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(112): Illuminate\Routing\ControllerDispatcher->call(Object(Backend\Classes\BackendController), Object(Illuminate\Routing\Route), 'run')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(114): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(69): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(Backend\Classes\BackendController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'run')
#21 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'Backend\\Classes...', 'run')
#22 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Router.php(708): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Router.php(710): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Router.php(675): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#37 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#40 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#41 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#42 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#43 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#46 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#49 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#52 /var/www/irtrip.com/irtrip-b2b/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#53 /var/www/irtrip.com/irtrip-b2b/index.php(44): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#54 {main}
```

**Comment by zamilnugraha** [comment]
@LukeTowers I have problem like @Shara96 and i via composer but does't work. can you help to solved this problem.??

**Comment by w20k** [comment]
@zamilnugraha post you error log & stack trace?

**Comment by seanthepottingshed** [comment]
@w20k 

This has randomly occurred to me in the past. Haven't been able to replicate since, so curious as to why this may happen.

**Comment by Balex70** [comment]
Got the same issue. The weird thing that it was working before, and in one day I got this error:
```
InvalidArgumentException: View [welcome] not found. in /var/www/tfs2/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php:137
Stack trace:
#0 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php(92): Illuminate\View\FileViewFinder->findInPaths('welcome', Array)
#1 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/View/FileViewFinder.php(76): Illuminate\View\FileViewFinder->findNamespacedView('rainlab.user::w...')
#2 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/View/Factory.php(128): Illuminate\View\FileViewFinder->find('rainlab.user::w...')
#3 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Mail/Mailer.php(328): Illuminate\View\Factory->make('rainlab.user::w...', Array)
#4 /var/www/tfs2/vendor/october/rain/src/Mail/Mailer.php(406): Illuminate\Mail\Mailer->renderView('rainlab.user::w...', Array)
#5 /var/www/tfs2/vendor/october/rain/src/Mail/Mailer.php(73): October\Rain\Mail\Mailer->addContent(Object(Illuminate\Mail\Message), 'rainlab.user::w...', NULL, NULL, Array)
#6 /var/www/tfs2/vendor/october/rain/src/Mail/Mailer.php(181): October\Rain\Mail\Mailer->send('rainlab.user::w...', Array, Object(Closure))
#7 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): October\Rain\Mail\Mailer->sendTo(Array, 'rainlab.user::w...', Array, Object(Closure))
#8 /var/www/tfs2/plugins/rainlab/notify/notifyrules/SendMailTemplateAction.php(55): Illuminate\Support\Facades\Facade::__callStatic('sendTo', Array)
#9 /var/www/tfs2/plugins/rainlab/notify/models/RuleAction.php(49): RainLab\Notify\NotifyRules\SendMailTemplateAction->triggerAction(Array)
#10 /var/www/tfs2/plugins/rainlab/notify/models/NotificationRule.php(81): Rainlab\Notify\Models\RuleAction->triggerAction(Array)
#11 /var/www/tfs2/plugins/rainlab/notify/classes/Notifier.php(108): RainLab\Notify\Models\NotificationRule->triggerRule()
#12 /var/www/tfs2/plugins/rainlab/notify/classes/EventParams.php(35): RainLab\Notify\Classes\Notifier->fireEvent('RainLab\\User\\No...', Array)
#13 [internal function]: RainLab\Notify\Classes\EventParams->handle()
#14 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(29): call_user_func_array(Array, Array)
#15 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(87): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#16 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(31): Illuminate\Container\BoundMethod::callBoundMethod(Object(October\Rain\Foundation\Application), Array, Object(Closure))
#17 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Container/Container.php(549): Illuminate\Container\BoundMethod::call(Object(October\Rain\Foundation\Application), Array, Array, NULL)
#18 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(94): Illuminate\Container\Container->call(Array)
#19 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(114): Illuminate\Bus\Dispatcher->Illuminate\Bus\{closure}(Object(RainLab\Notify\Classes\EventParams))
#20 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(RainLab\Notify\Classes\EventParams))
#21 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php(98): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#22 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php(49): Illuminate\Bus\Dispatcher->dispatchNow(Object(RainLab\Notify\Classes\EventParams), false)
#23 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php(76): Illuminate\Queue\CallQueuedHandler->call(Object(Illuminate\Queue\Jobs\SyncJob), Array)
#24 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Queue/SyncQueue.php(42): Illuminate\Queue\Jobs\Job->fire()
#25 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Queue/QueueManager.php(268): Illuminate\Queue\SyncQueue->push(Object(RainLab\Notify\Classes\EventParams))
#26 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): Illuminate\Queue\QueueManager->__call('push', Array)
#27 /var/www/tfs2/plugins/rainlab/notify/classes/Notifier.php(95): Illuminate\Support\Facades\Facade::__callStatic('push', Array)
#28 /var/www/tfs2/plugins/rainlab/notify/classes/Notifier.php(85): RainLab\Notify\Classes\Notifier->queueEvent('RainLab\\User\\No...', Array)
#29 [internal function]: RainLab\Notify\Classes\Notifier::RainLab\Notify\Classes\{closure}(Object(RainLab\User\Models\User), Array)
#30 /var/www/tfs2/vendor/october/rain/src/Events/Dispatcher.php(233): call_user_func_array(Object(Closure), Array)
#31 /var/www/tfs2/vendor/october/rain/src/Events/Dispatcher.php(197): October\Rain\Events\Dispatcher->dispatch('rainlab.user.re...', Array, false)
#32 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(221): October\Rain\Events\Dispatcher->fire('rainlab.user.re...', Array)
#33 /var/www/tfs2/plugins/rainlab/user/components/Account.php(259): Illuminate\Support\Facades\Facade::__callStatic('fire', Array)
#34 /var/www/tfs2/modules/cms/classes/ComponentBase.php(187): RainLab\User\Components\Account->onRegister()
#35 /var/www/tfs2/modules/cms/classes/Controller.php(873): Cms\Classes\ComponentBase->runAjaxHandler('onRegister')
#36 /var/www/tfs2/modules/cms/classes/Controller.php(740): Cms\Classes\Controller->runAjaxHandler('onRegister')
#37 /var/www/tfs2/modules/cms/classes/Controller.php(373): Cms\Classes\Controller->execAjaxHandlers()
#38 /var/www/tfs2/modules/cms/classes/Controller.php(224): Cms\Classes\Controller->runPage(Object(Cms\Classes\Page))
#39 /var/www/tfs2/modules/cms/classes/CmsController.php(50): Cms\Classes\Controller->run('/')
#40 [internal function]: Cms\Classes\CmsController->run('/')
#41 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): call_user_func_array(Array, Array)
#42 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\Routing\Controller->callAction('run', Array)
#43 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Route.php(212): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Cms\Classes\CmsController), 'run')
#44 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Route.php(169): Illuminate\Routing\Route->runController()
#45 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Router.php(658): Illuminate\Routing\Route->run()
#46 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#47 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#48 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#49 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#51 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#52 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(63): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#54 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#55 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#57 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#58 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(66): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#60 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#61 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#63 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Router.php(660): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#64 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#65 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Router.php(601): Illuminate\Routing\Router->runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))
#66 /var/www/tfs2/vendor/october/rain/src/Router/CoreRouter.php(20): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#67 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(176): October\Rain\Router\CoreRouter->dispatch(Object(Illuminate\Http\Request))
#68 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(30): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#69 /var/www/tfs2/plugins/bedard/debugbar/vendor/barryvdh/laravel-debugbar/src/Middleware/Debugbar.php(51): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#70 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#71 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#72 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(46): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#73 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(149): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#74 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php(53): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#75 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#76 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(151): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#77 /var/www/tfs2/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#78 /var/www/tfs2/index.php(43): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#79 {main}
```

**Comment by LukeTowers** [comment]
What plugins do you have installed @balex70?

**Comment by mohammadmojrian** [comment]
@Balex70 try this:
```
artisan cache:clear
artisan clear-compiled
composer dump
```

**Comment by Balex70** [comment]
@LukeTowers I have installed: RainLab.User, RainLab.UserPlus, RainLab.Location, RainLab.Notify
@mohammadmojrian The problem is that site was installed by another developer via installer, so I can't do "composer dump".
Anyway, looks like this error related with this https://github.com/rainlab/notify-plugin/issues/10 issue on Rainlab.Notify plugin. Method findInPaths can't find "welcome" template which is correct, because this template doesn't exist. When I make changes directly in Database for table rainlab_notify_rule_actions, column config_data={"mail_template":"rainlab.user::mail.activate","send_to_mode":"user"}, its working good. As you can see mail_template=rainlab.user::mail.activate instead rainlab.user::welcome. So again, it all realted to Rainlab.Notify plugin.

**Comment by dioneSky** [comment]
I had the same error, when I created a new dir in Controllers and put there some of my php file controllers, and then I regreat of that idea so I remove the directory and put back the files to controllers dir, since then I started facing the same error because de autload_classmap.php didnt update.

So I tryed this:

artisan cache:clear
artisan clear-compiled
composer dump

none of above trys worked.

so I follow the track error and i edited the file located in ..\your_project_name\vendor\composer\autload_classmap.php and change the right controllers dir.
